### PR TITLE
fix(logs): make logs great again

### PR DESCRIPTION
**Pull request by jkurz**
- check to make sure log line is long enough to manipulate. If not it's just a crap line and ignore it.

### CODE REVIEW

**Review by jritsema — CHANGES_REQUESTED** [review]
Thanks for fixing this!  Looks good.  

Oh, can you run `go fmt`?

**Comment by jkurz** [comment]
go fmt has already been run from what I can see.

File: cmd/logs.go [hunk 1]
```diff
@@ -82,27 +82,30 @@ func (slice Logs) Swap(i, j int) {
...
+						}
 					}
-				}
 
-				var logObject = logObject{}
+					var logObject = logObject{}
 
-				logObject.Time = timeValue
-				logObject.Log = strings.Join(line, " ")
+					logObject.Time = timeValue
```
Comment (jritsema) [inline cmd/logs.go:103]: @jkurz this is the whitespace change I was referring to.

Comment (jkurz) [reply cmd/logs.go:103]: I am indenting this line, because it's now in a conditional block.

Comment (jritsema) [reply cmd/logs.go:103]: Oh, I apologize, I misread the code thinking it was the whitespace issues we were seeing before.

**Review by jritsema — APPROVED** [review]

**Pull request merged**